wifinina/wifinina.go

```diff
@@ -126,6 +126,12 @@ type Device struct {
 	ip    uint32
 	port  uint16
 	mu    sync.Mutex
+
+	// ResetIsHigh controls if the RESET signal to the processor
+	// should be High or Low (the default). Set this to true
+	// before calling Configure() for boards such as the Arduino MKR 1010,
+	// where the reset signal needs to go high instead of low.
+	ResetIsHigh bool
```

Comment:
This does look to me as a hack/workaround.
Shall we not have `RESET_HIGH` and `RESET_LOW` pins then instead?
Alternatively `RESET_HIGH` and `RESET` for backwards compatibility.
Then users can choose which pin to assign and which leave as "nopin".
And logic in `Configure()` then decides which pin to use (can panic / error when both defined or none)

Reply:
Another alternative: control behaviour of `RESET` pin with a build tag.
`Arduino MKR 1010` target could have, say, `wifinina_reset_high` build tag. Next, wifinina driver has an extra file with this tag here that sets an internal flag for such behaviour.

Reply:
Both sound more complicated somehow.

Reply:
Second alternative is more complicated to implement perhaps, but easier and transparent to use.
To me it feels the reset pin behaviour is board/hardware specific and weird to have to think about it in the application code.

The first approach, I like it less myself, just brainstormed how can we do it differently.

Reply:
We have drivers that have different behaviour for different build tags already, see https://github.com/tinygo-org/drivers/tree/release/ws2812

Reply:
Yes, but they do more than just swap high/low for a specific pin. Adding a build tag for that seems like overkill, IMO.

Reply:
The change I made seemed to me like the "simplest thing that could possibly work" to address the swap of setting high/low.  Adding more code/logic just seemed like a lot extra for that.

Reply:
> To me it feels the reset pin behaviour is board/hardware specific and weird to have to think about it in the application code.

It is a single line you need to add if you are using one of only a couple of specific boards. Hence why I tried to make the least intrusive change possible.